charactersheet/charactersheet/viewmodels/dm/encounter_sections/environment_section.js

```diff
@@ -27,6 +28,25 @@ function EnvironmentSectionViewModel(parentEncounter) {
 
     //Public Methods
 
+    self.exhibitEnvironment = function() {
```

Comment:
Can you change this name to `toggleExhibit`? Makes it more clear that the value is being changed rather than just set.